### DISCUSSION TITLE
Cannot use bonsai together with camera because PointGrey camera can be used with SDK version 3, but not support by SDK 1.29

### DISCUSSION THREAD

**Discussion by KickStart10032** [bonsai-rx/bonsai#1547] [Q&A]
I got a new FLIR BFS camera and want to use it with Bonsai. I found that the camera works with Spinnaker SDK 3, but not with 1.9 version. However, Bonsai works with Spinnaker SDK 1.9, but not 3. I am wondering if there is get around solution that I could use the camera with Bonsai? Thanks!

**Comment by bruno-f-cruz** [comment]
Heya!
While it has been discussed, there is currently no roadmap for this upgrade to the package to the new SDK versions. Meanwhile, there seems to be an [open PR](https://github.com/bonsai-rx/spinnaker/pull/7) to the package that targets the 3.x.x version of the SDK. If you are in a pinch, I would definitely give it a try!



Cheers,
Bruno
